public/index.html

```diff
@@ -177,7 +216,8 @@
       }
     </script>
 
-    <div id="root" class="container"></div>
+    <!-- <div id="root" class="container"></div> -->
+    <div id="as" class="container"></div>
```

Comment:
We should probably rename id of this two into something meaningful. Eg: 
 `<div id="countryPerformace" class="container">`
`<div id="asPerformance" class="container">`